Dockerfile.compile

```diff
@@ -1,6 +1,12 @@
-FROM gcr.io/spinnaker-marketplace/gradle_cache
-MAINTAINER delivery-engineering@netflix.com
-ENV GRADLE_USER_HOME /gradle_cache/.gradle
-COPY . compiled_sources
-WORKDIR compiled_sources
-RUN ./gradlew --no-daemon echo-web:installDist -x test
\ No newline at end of file
+FROM ubuntu:eoan
+MAINTAINER sig-platform@spinnaker.io
+
+RUN apt-get update \
+    && apt-get -y upgrade \
+    && apt-get install -y \
+        openjdk-8-jdk-headless \
+    && apt-get clean
```

Comment:
In the past, we've had flaky container builds doing an `apt upgrade`, not to mention that it takes a relatively long time. Since this is only an intermediate container, what do you think about simply basing it `FROM openjdk8`, just like `Dockerfile.java8`?

Reply:
Ooh, yeah, I forgot about those. That's way better, thanks. I'll work on updating all the PRs, make sure they still build, and then update this comment when I'm done.

Reply:
They all compiled fine with `openjdk:8`, so I've updated all 10 of the PRs whenever you get a chance to review, thanks.